src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11784,7 +11784,7 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
             // Required load is beyond the max system capacity
 
             Q_h_TU_PL = CompEvaporatingCAPSpdMax;
-            TU_HeatingLoad = CompEvaporatingCAPSpdMax;
+            // TU_HeatingLoad = CompEvaporatingCAPSpdMax;
```

Comment:
I am not very familiar with this model, so I will ask that this be explained. Why is this commented out?

Reply:
I'm still working on this part. It seems the capacity calculation here for CompEvaporatingCAPSpdMax and at the compressor speed calculation part in `VRFOU_CalcCompH(.)` are different. The following is a plot comparing the max-speed capacity at these two locations for January 1st. The red one is the CompEvaporatingCAPSpdMax here and the blue one is the max-speed capacity in `VRFOU_CalcCompH(.)`. Since VRFOU_CalcCompH(.) uses this TU_HeatingLoad as an input representing the load to be met (which is trimmed by a low capacity), the compressor speed is set to very low. Logically it seems weird to use a trimmed load to calculate compressor speed as the function already takes care of the case where load > capacity. So I commented it out for now but I'm not certain if this is the final solution. It might need some further adjustment as the Piping loss calculation might be affected.
<img width="468" alt="image" src="https://github.com/NREL/EnergyPlus/assets/13421161/b9caf5c3-83ab-496a-afca-27ee9f7d3b78">

The max-speed capacities are different at these two places (for the same timestep) because the two temperature inputs are different. They both use a curve like this 
`this->CoffEvapCap * this->RatedEvapCapacity * CurveValue(state, this->OUCoolingCAPFT(NumOfCompSpdInput), arg1, arg2)`. 

The two temperature args are like this

  | TU_limiting | Compressor spd calc
-- | -- | --
Arg 1 name | T_discharge | This->IUCondensingTemp
Arg 1 value | saturation temperature of a refrigerant, a function of   pressure | T_coil_in + (Tout – T_coil_in) / (1 – BFH) + DeltaT
Arg 2 name | T_suction | CapMaxTe
Arg 2 value | T_coil_surf – deltaT. deltaT is a function of outdoor unit   superheating degrees (this->SH) | OutDryBulb – this->SH

The values of the two temperature arguments are like these in 1/1
![arg1](https://github.com/NREL/EnergyPlus/assets/13421161/3585af20-9129-4f0b-bd31-bff69ccdad2c)
![arg2](https://github.com/NREL/EnergyPlus/assets/13421161/120312bd-73ef-4bd7-9eb4-a1b976387de9)

Reply:
I think some detailed plots of a zone TU when the OA temp drops dramatically in the same day will show whether this is working correctly or not. The other thing is to check on the PipingCorrection* calculation/reports to make sure those make sense.